kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/sequences/SequenceMatchersTest.kt

```diff
@@ -4,12 +4,20 @@ import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.core.spec.style.StringSpec
 import io.kotest.matchers.sequences.shouldContainExactly
 import io.kotest.matchers.sequences.shouldHaveSingleElement
+import io.kotest.matchers.sequences.shouldNotBeEmpty
 import io.kotest.matchers.sequences.shouldNotContainExactly
 import io.kotest.matchers.sequences.shouldNotHaveSingleElement
 import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.isActive
+import kotlin.time.Duration.Companion.milliseconds
 
 class SequenceMatchersTest : StringSpec({
 
+   "beEmpty does not generate the entire sequence".config(timeout = 500.milliseconds) {
+      val infinite = sequence { while (isActive) yield(1) }
```

Comment:
Should this test ever fail, it would take 500 ms to do so. The following fails faster:
```kotlin
val canConsumeOneElement = sequence { yield(1); throw Exception("should not get second element") }
```
WDYT

Reply:
Changed as suggested